src/pybind/mgr/volumes/fs/nfs.py

```diff
@@ -708,3 +709,48 @@ def list_nfs_cluster(self):
         except Exception as e:
             log.exception("Failed to list NFS Cluster")
             return getattr(e, 'errno', -1), "", str(e)
+
+    def _show_nfs_cluster_info(self, cluster_id):
+        self._set_cluster_id(cluster_id)
+        completion = self.mgr.list_daemons(daemon_type='nfs')
+        self.mgr._orchestrator_wait([completion])
+        orchestrator.raise_if_exception(completion)
+        host_ip = {}
+        # Here completion.result is a list DaemonDescription objects
+        for cluster in completion.result:
+            if self.cluster_id == cluster.service_id():
+                """
+                getaddrinfo sample output: [(<AddressFamily.AF_INET: 2>,
+                <SocketKind.SOCK_STREAM: 1>, 6, 'xyz', ('172.217.166.98',2049)),
+                (<AddressFamily.AF_INET6: 10>, <SocketKind.SOCK_STREAM: 1>, 6, '',
+                ('2404:6800:4009:80d::200e', 2049, 0, 0))]
+                """
+                try:
+                    host_ip[cluster.daemon_id] = {
```

Comment:
on second thought, is the daemon_id useful in this context? maybe return a List instead?

Reply:
Right daemon_id is not instead I have added cluster_id
```
{
    "vstart": [
        {
            "hostname": "varsha",
            "IP": [
                "::1",
                "127.0.0.1"
            ],
            "Port": 2049
        }
    ]
}
```